### ISSUE TITLE
ViewModelForwarding: false positive when using a VM as key for LaunchedEffect?

### ISSUE THREAD

**Issue by leinardi** [twitter/compose-rules#47]
**Describe the bug**
I'm not 100% sure but I think I'm getting a false positive if I use a VM as key of a `LaunchedEffect` or `DisposableEffect`.

**To Reproduce**
```kotlin
@Composable
fun MyComposable(viewModel: MyComposableViewModel = hiltViewModel()) {
    LaunchedEffect(viewModel) {
        viewModel.onUiEvent(Event.OnViewAttached)
    }
}
```

**Expected behavior**
No `ViewModelForwarding` violation is raised when using a VM as parameter for `LaunchedEffect()` or `DisposableEffect`.

**Additional context**
I'm getting this with v0.0.5 (the latest v0.0.6 is not available on maven yet).

**Comment by mrmans0n** [comment]
Yep, you are right, we should probably filter out *Effect(...) and remember*(...) calls as well.
